mllib/src/test/scala/org/apache/spark/mllib/tree/DecisionTreeSuite.scala

```diff
@@ -852,10 +399,7 @@ class DecisionTreeSuite extends SparkFunSuite with MLlibTestSparkContext {
     }
 
     // test when no valid split can be found
-    val rootNode = model.topNode
-
-    val gain = rootNode.stats.get
-    assert(gain == InformationGainStats.invalidInformationGainStats)
```

Comment:
Does the check on L380 make sense anymore? It looks like we never use `InformationGainStats.invalidInformationGainStats` anymore so we could remove it altogether?